RunCMake.sh

```diff
@@ -92,8 +92,14 @@ verlt() {
 }
 
 if [ "$Distribution" = "Ubuntu" -a "$Version" = "22" ] || [ "$Distribution" = "Ubuntu" -a "$Version" = "24" ] || [ "$Distribution" = "Mint" -a "$Version" = "21" ]; then
-  packages=(cmake build-essential ninja-build libxrandr-dev libxinerama-dev libomp-dev libxcursor-dev libxi-dev uuid-dev mold libfreetype-dev libtinfo5 libomp-dev)
```

Comment:
You did remove libomp-dev. Is this no longer needed when building with gcc on Ubuntu 22?

Reply:
The package is still present further down under clang packages as `libomp-14-dev`. We need to use the version that matches the clang version. Using `libomp-dev` points to the latest version which conflicts with `libomp-14-dev` as there can be only one version installed apparently.